### PR TITLE
Feat/tldr: refactor and introduce context menu app

### PR DESCRIPTION
**Pull request by stefanpeknik**

### CODE REVIEW

**Comment by stefanpeknik** [comment]
explanation behind context menu: https://github.com/Rapptz/discord.py/issues/7823

File: frontroomsbot/cogs/tldr.py [hunk 1]
```diff
@@ -1,79 +1,214 @@
...
+
+class StartMsgOlderThanEndMsgError(TldrError):
+    def __init__(
+        self,
+        error_msg: str = "Starting message must not be newer than the ending message.",
+    ):
+        super().__init__(error_msg)
+
+
 class TldrCog(commands.Cog):
```
Comment (S1ro1) [inline frontroomsbot/cogs/tldr.py]: this could be done via ConfigCog, all of these consts could be configured via that

**Review by S1ro1 — COMMENTED** [review]
Add functionality to tldr one message, else lgtm

**Review by S1ro1 — APPROVED** [review]
lgtm

**Pull request merged**